lib/components/DateTime/Calendar.module.scss

```diff
@@ -120,12 +114,15 @@ $line-height-row: $calendar-column-width;
         margin-left: $grid-size;
 
         & > button.selected {
-            color: var(--color-text-active);
-            background-color: var(--color-bg-btn-primary-rest);
+            color: var(--color-accent-foreground);
+            background-color: var(--color-accent-selected);
+
+            &:hover, &:focus {
+                background-color: var(--color-accent-selected-hover);

```

Comment:
Button.scss just gets color-accent-hover for hover and focus, why is this selected-hover?

Can you go through the active/hover/focus stylings all-up and make them consistent? Ideally, these should all come through the same class/mixin

Reply:
This is different than button.scss.... the calendar has a concept of a `selected` button.. which is the day that is selected, therefore this has to have it's own hover/focus styles... that's why it uses selected hover instead of hover.

I have gone through all the active/hover/focus styles and they are all consistent ... we should be able to bug bash once this is ready to go to renderer, but you can play around [here](https://patobeltran.github.io/custom-iot-baseline/) if you want to make sure before merging this